### DISCUSSION TITLE
Where to request an update to a package?

### DISCUSSION THREAD

**Discussion by xavier2k6** [chocolatey/choco#3429] [Q&A] [answered]
Chocolatey is currently distributing [jom 1.1.2](https://community.chocolatey.org/packages/jom)

There's a new version of jom available at
https://download.qt.io/official_releases/jom/jom_1_1_4.zip
https://mirrors.ukfast.co.uk/sites/qt.io/official_releases/jom/changelog.txt

**Comment by pauby** [comment, marked as answer]
Please follow the [triage process](https://docs.chocolatey.org/en-us/community-repository/users/package-triage-process).

Packages are maintained by volunteer maintainers and software vendors / authors. Chocolatey does not maintain packages.
